packages/components/psammead-image/src/index.amp.jsx

```diff
@@ -1,13 +1,23 @@
 import React from 'react';
 import { number, string } from 'prop-types';
 
-const AmpImg = ({ alt, attribution, layout, src, srcset, height, width }) => {
+const AmpImg = ({
```

Comment:
Instead of `unpacking` props and then `packing` them again, Could we have
```jsx
const AmpImg = props => {
  const { alt /* the rest */ } = props
}
```
Looks much cleaner IMO.

Reply:
Or simplify to 
```jsx
const AmpImg = props => {
   const { srcset } = props;
   return <amp-img {...props} srcSet={srcset} />
}
```

Reply:
> Or simplify to
> 
> ```js-jsx
> const AmpImg = props => {
>    const { srcset } = props;
>    return <amp-img {...props} srcSet={srcset} />
> }
> ```

Probably shouldnt pass the `srcset` prop, could do
```jsx
const { srcset, ...otherProps } = props;
return <amp-img {...otherProps} srcSet={srcset} />
```

Reply:
`prop-types` doesn't guard against passing extra props that are not defined. You can pass in any extra prop that is not defined and it wouldn't even raise a warning.